Plugin/CartIntercept.php

```diff
@@ -56,37 +55,37 @@ public function _gate(Cart $cart)
             return $cart;
         }
     }
+
     public function sendCart(Cart $cart, $storeId)
     {
         $customer = $cart->getCustomerSession()->getCustomer();
         $email = $customer->getEmail();
-        $this->client = $this->client->getClient(true, $storeId);
+        $client = $this->clientManager->getClient($storeId);
         if ($email || $anonymousEmail = $this->isAnonymousReady($storeId)) {
             $items = $this->_getItems($cart);
             $data = [
-                'email'             => $email,
-                'items'             => $items,
-                'incomplete'        => 1,
-                'message_id'        => $this->sailthruCookie->getBid(),
-            ];    
-            $email = $email ? $email : $anonymousEmail;
```

Comment:
@lindsaycooper @silverman63 This line is unused so it's safe to remove, but I'm wondering what it was supposed to be doing in the first place. This question doesn't block this PR, but we need to discuss this.